tests/integration/models/reddit/test_message.py

```diff
@@ -98,6 +98,22 @@ def test_reply(self, _):
             assert reply.body == "Message reply"
             assert reply.first_message_name == message.fullname
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_unblock_subreddit(self, _):
+        self.reddit.read_only = False
+        with self.use_cassette():
+            message1 = next(self.reddit.inbox.messages(limit=1))
+            assert isinstance(message1, SubredditMessage)
+            message_fullname = message1.fullname
+            message1.block()
+            message2 = next(self.reddit.inbox.messages(limit=1))
+            assert message2.fullname == message_fullname
+            assert message2.subject == "[message from blocked subreddit]"
+            message2.unblock_subreddit()
+            message3 = next(self.reddit.inbox.messages(limit=1))
+            assert message3.fullname == message_fullname
+            assert message3.subject != "[message from blocked subreddit]"
+
```

Comment:
What happens if you unblock a subreddit that isn't blocked?

Reply:
You get a 200, your x-ratelimit-used goes up by one, and your x-ratelimit-remaining goes down by one. Other than that, nothing, as it's not a toggle the way /api/wiki/hide is. The same thing is true if you try to unblock multiple subs at once. If you use a real t5 or r9 you get a 500.

Reply:
Do you want me to add a test for this? It doesn't appear to do anything, but I could test it anyway.

Reply:
No we don't need to for that.